doc/releases/release-notes-2.2.rst

```diff
@@ -193,6 +205,7 @@ Boards & SoC Support
    * Google Kukui EC
    * NXP i.MX RT1010 Evaluation Kit
    * Silicon Labs EFM32 Giant Gecko GG11
+   * Silicon Labs EFM32 Jake Gecko
```

Comment:
I believe EFM32**JG**12B stands for **Jade** Gecko.

Reply:
It does, I was fixing this while you commented.

Reply:
thanks anyways